### PR TITLE
ISPN-11488 documentation edits and corrections

### PR DESCRIPTION
**Pull request by oraNod**
https://issues.redhat.com/browse/ISPN-11488

needs backport to 1.1.x

### CODE REVIEW

File: documentation/asciidoc/topics/proc_creating_external_services.adoc [hunk 1]
```diff
@@ -1,29 +1,53 @@
...
-To connect to {brandname} clusters from clients running outside {k8s}, create
-external services with {ispn_operator}.
+= Exposing {brandname} to External Clients
+Expose {brandname} clusters to clients running outside {k8s} with external
+services.
 
 .Procedure
 
-. Modify your custom resource definition to configure an external service and then apply the changes.
+* Modify your custom resource definition to configure an external service and then apply the changes.
```
Comment (Crumby) [inline documentation/asciidoc/topics/proc_creating_external_services.adoc]: User can't modify custom resource definition. What modify your Infinispan resource?

Comment (oraNod) [reply documentation/asciidoc/topics/proc_creating_external_services.adoc]: @Crumby thanks Pavel. I've made a bunch of changes to clarify the wording here. it probably expands the scope of the PR a bit but we might as well apply these changes across the board.

File: documentation/asciidoc/topics/proc_configure_logging.adoc [hunk 2]
```diff
@@ -3,7 +3,8 @@
 
 .Procedure
 
-. Modify your custom resource definition to include logging resources and then apply the changes.
+. Specify logging configuration with the `spec.logging` CR object and then
```
Comment (Crumby) [inline documentation/asciidoc/topics/proc_configure_logging.adoc]: _~~CR object~~_ or _in your Infinispan CR_. There is nothing like `spec.logging` CR object.

File: documentation/asciidoc/topics/proc_create_cluster_auth.adoc [hunk 3]
```diff
@@ -16,7 +15,8 @@ link:#ref_auth_secret-auth[Authentication Secrets].
 $ {oc_apply_cr} connect_secret.yaml
 ----
 +
-. Modify your custom resource definition to include the authentication secret and then apply the changes.
+. Specify the authentication secret with the `spec.security.endpointSecretName`
+CR object and then apply the changes.
```
Comment (Crumby) [inline documentation/asciidoc/topics/proc_create_cluster_auth.adoc]: Same as above. There is nothing like `spec.security.endpointSecretName` CR object

File: documentation/asciidoc/topics/proc_creating_routes.adoc [hunk 4]
```diff
@@ -1,29 +1,30 @@
 [id='creating_routes-{context}']
 = Creating Routes to Services
-In some cases, such as running local {k8s} clusters, you need to create routes
-to access {brandname} clusters from clients running outside {k8s}.
+For services of type `NodePort`, you might also need to create routes to access
```
Comment (Crumby) [inline documentation/asciidoc/topics/proc_creating_routes.adoc]: Creating routes doesn't have anything to do with `NodePort` services. For `Routes` in OpenShift you can directly reference default service. No need to create external one.

File: documentation/asciidoc/topics/proc_creating_routes.adoc [hunk 5]
```diff
@@ -1,29 +1,30 @@
...
 .Procedure
 
-* If you are not using TLS encryption, expose the default service as in the
-following example:
+* Create a route with `passthrough` encryption as in the following example:
 +
 [source,options="nowrap",subs=attributes+]
 ----
-$ oc expose svc/{example_crd_name}
+$ oc create route passthrough --service={example_crd_name}
```
Comment (Crumby) [inline documentation/asciidoc/topics/proc_creating_routes.adoc]: This requires cluster endpoints to be encrypted.

**Comment by ryanemerson** [comment]
I've only looked at the latest commit to change the default config, but looks good :+1:

**Comment by oraNod** [comment]
> I've only looked at the latest commit to change the default config, but looks good +1

@ryanemerson thanks! I've discussed with Vittorio and Pavel and we've decided to use this PR for some of the feedback that Pavel is providing during review.

**Comment by rigazilla** [comment]
merged, thanks @oraNod !

**Pull request merged**